README.md

```diff
@@ -1,79 +1,86 @@
+> Looks great  with light and dark color schemes. Shown here with [Arc](https://github.com/PapirusDevelopmentTeam/arc-kde) and [Arc-Dark](https://github.com/PapirusDevelopmentTeam/arc-kde) color schemes.
 
-Glitch-b-gone with Breezeway magick!
 
-![](low_contrast.png)
+| The usual glitch on dark themes            | ![](sierrabreeze.png) |
+| ------------------------------------------ | :-------------------: |
+| Glitch be gone with BreezewayMod. *Magic!* | ![](breezewaymod.png) |
+| Glitch be gone with BreezewayMod. *Magic!* | ![](low_contrast.png) |
 
-If overlapping windows have not enough contrast between each other it's sometimes hard to see overlapping titlebars of you use a flat theme. Luckily, BreezewayMod comes with a neat highlight line across the top of your titlebar to get that sorted out!
+On overlapping windows with not enough contrast between each other, it's hard to see overlapping titlebars if using a flat theme. Luckily, **BreezewayMod** comes with a neat highlight line across the top of titlebars to get that sorted out:
 
 ![](gradients.png)
 
-If you wish to have subtle gradients with a nicely drawn separator for your titlebar, you can have just that. The gradient and separator will, like all nifty color features, use the title bar color to determine their values. How cool is that?
+> **Note**: if you wish to have subtle gradients with a nicely drawn separator for titlebars, you can have just that. The gradient and separator will, like all nifty color features, use the title bar color to determine their values. *How cool is that?*
 
 ## Experimental Features
+
+With the power of Halloween and the dark arts combined, we can now match the titlebars color to the windows color... And I don't mean some lame shit where I make use of `QPalette::Window` in the hopes of getting a nice looking window color... Hmmm, no, no no!
+
 ![](match_colors.png)
 
-With the power of Halloween and the dark arts combined, we can now match the titlebar color to the window color. And I don't mean some lame shit where I make use of ```QPalette::Window``` in the hopes of getting a nice looking window color, mmh no no no! This feature is still in early beta though and color glitches _will_ occur. Uncheck "Match colors" in the settings to turn the feature off.
+> **Note**: this feature is still in early beta though and color glitches *will* occur. Uncheck **Match colors** in Plasma Settings to turn this feature off.
 
-## Installation
-Clone the repository to a directory of your choice:
-```
-git clone https://github.com/n4n0GH/breezeway-mod
-```
-Note: it is recommended to clone it somewhere where you can leave the built directory in case you want to uninstall the theme. If you accidentally deleted the directory, check below for how to uninstall manually.
+## Building and Installation
 
-### Dependencies
+Before preceding with the actual install process, make sure you have the following dependencies installed (for Ubuntu 18.04 and derivates):
```

Comment:
The differentiation between operating systems was intended to make it easy to understand how to procure the packages, i.e. through apt or similar. I need to write up an example for this, but we can keep it like suggested for now.

Reply:
Something like distro agnostic? I've delete it.